src/vaults/BaseVaults.sol

```diff
@@ -178,6 +188,12 @@ abstract contract BaseVault is Auth, Recoverable, IBaseVault {
         assets = manager.convertToAssets(address(this), shares);
     }
 
+    // --- Legacy views ---
+    /// @inheritdoc IBaseVault
+    function escrow() public view returns (address) {
```

Comment:
Doesn't this call need to be in `BaseInvestmentManager`? Since the v2.1 vaults call `manager.escrow()`